media/stump-volume-control/volume-control.lisp

```diff
@@ -15,4 +15,9 @@
   (let ((muted (search "[off]"
                        (run-shell-command (format nil "amixer -c ~d sset Master playback toggle" *sound-card*)
                                           t))))
+    (unless muted
```

Comment:
My impression is that this should be in a custom for for yourself, and not part of the contrib repository.

Reply:
Hm, but it is on all systems that use the dreaded PulseAudio, and "Speaker" is the general default name. But the command shouldn't be fired if not necessary, I think I'll add a variable *use-pulseaudio-hack* with a default of NIL.

Reply:
This is no customizable, see new commits (and comments + documentation).